schema-supplement.rst

```diff
@@ -52,40 +57,18 @@ instead of, or in addition to, the canonical package name.
 :attribute:`license`
 --------------------
 
-:Type: Special
+:Type: :type:`string`
 :Applies To: :object:`package`, :object:`component`
 
 Specifies the license or licenses
 under which the package is distributed.
-A :type:`string` value shall be used for a single license.
-If multiple licenses need to be specified,
-a :type:`list` shall be used.
-Each :type:`string` value of the :type:`list`
-is a license which *always* applies to the package.
-If a package is wholly or partly multi-licensed
-(that is, the user has a choice of license),
-the top level :type:`list` shall contain a :type:`list` of values
-representing possible licenses.
-For example, the value ``["CC-BY-4.0", ["GPL-2.0", "LGPL-3.0+"]]``
-indicates that some portions of the package
-are licensed under :string:`"CC-BY-4.0"`,
-while others are licensed as (at the user's choice)
-either :string:`"GPL-2.0"` or :string:`"LGPL-3.0+"`.
-If necessary, further nesting may be employed;
-each nesting level alternates
-between inclusive ("A **and** B")
-and exclusive ("A **or** B") licensing.
-
+The value shall be a well formed
+|SPDX|_ `License Expression`_ .
```

Comment:
Weak suggestion for your consideration:
```suggestion
|SPDX|_ `License Expression`_ .

```
(Just making the following text a new paragraph.)